addons/armaos/functions/fnc_shell_executeFile.sqf

```diff
@@ -0,0 +1,42 @@
+/**
+ * Executes a file.
+ * 
+ * Arguments:
+ * 0: Computer <OBJECT>
+ * 1: File pointer [<STRING>]
+ * 2: Path to file <STRING>
+ * 3: Options <LIST>
+ *
+ * Returns:
+ * Result of the file.
+ */
+
+params['_computer', '_path', '_options'];
+
+private _pointer = _computer getVariable "AE3_filepointer";
+private _filesystem = _computer getVariable "AE3_filesystem";
+
+private _terminal = _computer getVariable "AE3_terminal";
+private _username = _terminal get "AE3_terminalLoginUser";
+
+private _result = [format ["Command '%1' not found.", _path]];
+
+try
+{
+	_content = [_pointer, _filesystem, _path, _username, 0] call AE3_filesystem_fnc_getFile;
+
+	if(_content isEqualType {}) then
+	{
+		_result = [_computer, _options] call _content;
+
+		if(isNil "_result") exitWith {_result = [""]};
+
+		if(_result isEqualType []) exitWith {};
```

Comment:
I think we need a result in this case, like in the other cases, for example:

`if(_result isEqualType []) exitWith {_result = [""]};`

Reply:
exitWith in this case only breaks out of the try catch context.
The result is set in line 30 and does not to be modified if its a list.

Reply:
Ok. I see. By using `exitWith` you only leave the `try` scope. So `_result` is already set and will be returned in lin 42.